submitted_models/emesent_hovermap_sensor_config_1/model.sdf

```diff
@@ -0,0 +1,774 @@
+          <spring_stiffness>0</spring_stiffness>
+        </dynamics>
+        <use_parent_model_frame>0</use_parent_model_frame>
+      </axis>
+    </joint>
+    <link name='gimbal_tilt'>
+      <pose>0.218 0 0.125 0 -0 0</pose>
+      <inertial>
+        <pose>-2.4e-05 0.002445 -2e-05 0 -0 0</pose>
+        <mass>0.0001</mass>
+        <inertia>
+          <ixx>0.00216206</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>0.00106169</iyy>
+          <iyz>0</iyz>
+          <izz>0.00218206</izz>
+        </inertia>
+      </inertial>
+      <collision name='gimbal_tilt_collision'>
+        <pose>0 0 0 0 -0 0</pose>
+        <geometry>
+          <box>
+            <size>0.001 0.001 0.001</size>
+          </box>
+        </geometry>
+        <surface>
+          <contact>
+            <ode/>
+          </contact>
+          <friction>
+            <ode/>
+          </friction>
+        </surface>
+      </collision>
+      <visual name='gimbal_tilt_visual'>
+        <pose>0 0 0 0 -0 0</pose>
+        <geometry>
+          <mesh>
+            <scale>1 1 1</scale>
+            <uri>meshes/gimbal_tilt.dae</uri>
+          </mesh>
+        </geometry>
+      </visual>
+      <visual name='gimbal_tilt_fixed_joint_lump__gimbal_flashlight_visual_2'>
+        <pose>-0.012 -0.025 -0.01 0 0 -1.57</pose>
+        <geometry>
+          <mesh>
+            <scale>0.005 0.005 0.005</scale>
+            <uri>meshes/spotlight.dae</uri>
+          </mesh>
+        </geometry>
+      </visual>
+      <velocity_decay/>
+      <velocity_decay/>
+      <sensor name='gimbal_camera' type='camera'>
+        <update_rate>10</update_rate>
+        <camera name='gimbal_camera_sensor'>
+          <horizontal_fov>1.287</horizontal_fov>
+          <image>
+            <width>300</width>
+            <height>300</height>
+          </image>
+          <clip>
+            <near>0.02</near>
+            <far>300</far>
+          </clip>
+          <noise>
+            <type>gaussian</type>
+            <mean>0</mean>
+            <stddev>0.007</stddev>
+          </noise>
+        </camera>
+        <pose>0 0.005 0.002 0 -0 0</pose>
+      </sensor>
+      <gravity>1</gravity>
+      <velocity_decay/>
+      <light name='light_source' type='spot'>
+        <pose>0.03 -0.02 -0.01 0 -1.57 0</pose>
+        <attenuation>
+          <range>40</range>
+          <linear>0</linear>
+          <constant>0.02</constant>
+          <quadratic>0.02</quadratic>
+        </attenuation>
+        <diffuse>220 210 175 1</diffuse>
+        <specular>220 210 175 1</specular>
+        <spot>
+          <inner_angle>0.7</inner_angle>
+          <outer_angle>1</outer_angle>
+          <falloff>1</falloff>
+        </spot>
+        <cast_shadows>1</cast_shadows>
+        <direction>0 0 -1</direction>
+      </light>
+      <sensor name='gimbal_imu' type='imu'>
+        <always_on>1</always_on>
+        <update_rate>100</update_rate>
```

Comment:
This `imu` is missing noise parameters.

Reply:
This imu is not really designed to be exposed to users. 

Its output is used internally by the gimbal plugin to provide a stabilised gimbal. This recreates the interface we work with on the systems track, where we use BASECAM gimbal controller for stabilisation, and provide our own control velocity inputs which is the interface provided by the gimbalControlPlugin.
let me know if you still want this exposed.

Reply:
default noise parameters added